Joesdev/Exposure-API
# SD-31 Delete Hierarchy

## Whats This PR Do? 
- Adds a delete route
- Adds a destroy function
- Adds two test functions for destroy

## Steps To Review
- **Register a new user** to login into the application
- **Navigate to /hierarchy/create**
- **Fill out and submit** the form (Input Ex: Get Over My Spider Phobia)
- _Now a Hierarchy has been created, the goal of the task is to delete this hierarchy_
- **Navigate to /hierarchy/10/edit**
- **Click the Delete button**
- **View the Hierarchies table** in the database(phmyadmin), **verify** the **row** containing an id of 10 and a goal of the string you typed **no longer exists**.
